BankLibrary/Bank.cs

```diff
@@ -5,22 +5,104 @@ namespace BankLibrary
 {
     public class Bank<T> where T : Account
     {
-        private readonly List<Account> _accounts = new();
+        private const string KgkPassPhrase = "CleanUp";
+        private Dictionary<Locker,object> _lockers = new();
+        private readonly List<T> _accounts = new();
+        public int AddLocker(string keyword, object data)
+        {
+            var locker = new Locker(_lockers.Count + 1, keyword, data);
+            _lockers.Add(locker, data);
+            return locker.Id;
+        }
+
+        public object GetLockerData(int id, string keyword)
+        {
+            var tempLocker = new Locker(id, keyword);
```

Comment:
я бы назвал просто `key`
```suggestion
            var key = new Locker(id, keyword);
```